src/__tests__/index.js

```diff
@@ -750,7 +750,20 @@ test(
         (var(--fluid-screen) - ((var(--fluid-min-width) / 16) * 1rem)) /
         ((var(--fluid-max-width) / 16) - (var(--fluid-min-width) / 16))
     )`,
-    'calc((var(--fluid-screen) - var(--fluid-min-width)/16*1rem)/(var(--fluid-max-width)/16 - var(--fluid-min-width)/16))'
+    'calc((var(--fluid-screen) - ((var(--fluid-min-width)/16)*1rem))/(var(--fluid-max-width)/16 - var(--fluid-min-width)/16))'
+  )
+);
+
+test(
+  'should preserve division precedence (3)',
+  testValue('calc(1/(10/var(--dot-size)))', 'calc(1/(10/var(--dot-size)))')
```

Comment:
This is the issue in https://github.com/cssnano/cssnano/issues/1235. Removing the brackets will give a different result if `--dot-size` contains any of the four operations.